### DISCUSSION TITLE
SSL Traffic

### DISCUSSION THREAD

**Discussion by gajeevan** [nginx-proxy/nginx-proxy#1946] [Q&A]
I currently have two nginx-proxy containers running. One visible to host port 80 "nginx80" and the other visible to host port 443 "nginx443".

I have the ssl configured and the website is working via ssl. The redirect from http to https is working but when I do a tcpdump port 443 on the container running the website I have no traffic.

It seems like the traffic is coming via port 80. Did I miss configure anything? Did I miss to expose a variable? Can someone kindly help. 

Thanks in advance!

**Comment by md5** [comment]
You're not going to be able to do this. If you have two `nginx-proxy` instances connecting to the same backend containers, it's not possible for the two proxies to connect to different ports.

**Comment by gajeevan** [comment]
When I run the following container. It crashes my server. The CPU goes to 100%

docker run -d -it \
--name nginx \
-h nginx \
-p 80:80 \
-p 443:443 \
-v $HOME/nginx/certs:/etc/nginx/certs:ro \
-v $HOME/nginx/vhost.d:/etc/nginx/vhost.d:ro \
-v $HOME/nginx/passwords:/etc/nginx/passwords:ro \
-v /var/run/docker.sock:/tmp/docker.sock:ro \
jwilder/nginx-proxy \
